src/applications/claims-status/components/ClaimsContentionList.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import { setFocus } from '../utils/page';
+
+const MAX_CONTENTIONS = 3;
+
+function ClaimsContentionList({ contentions, contentionsHeader }) {
+  const ellipsis = useRef(null);
```

Comment:
I think instead of using refs here we should try to use state management features (like `useState`) to update the UI more reactively. I would imagine that we would maybe just have a `useState`-managed flag that is controlled by the 'Show full list' button